### ISSUE TITLE
where main.py

### ISSUE THREAD

**Issue by Dave-SUN** [Li-ZK/MAFN-2021#1]
Excuse me, why didn't I find main.py in the file. Thank you very much

**Comment by Li-ZK** [comment]
Run dataset_data.py for each dataset. For example, ksc_data.py for ksc dataset.

在 2021-06-07 16:18:59，Dave-SUN ***@***.***> 写道：

Excuse me, why didn't I find main.py in the file. Thank you very much

**Comment by Dave-SUN** [comment]
Thank you very much.Could you please share ksc_30 and houston_30 with me？ Thank you

**Comment by Li-ZK** [comment]
You can use the PCA algorithm to generate the corresponding data sets.






发件人：Dave-SUN ***@***.***>
发送日期：2021-06-08 10:44:37
收件人：Li-ZK/MAFN-2021 ***@***.***>
抄送人：LZK ***@***.***>,Comment ***@***.***>
主题：Re: [Li-ZK/MAFN-2021] where main.py (#1)

Thank you very much.Could you please share ksc_30 and houston_30 with me？ Thank you

**Comment by Dave-SUN** [comment]
Yes.I have obtained the data sets after PCA by Matlab,but when I run the program,there are bugs.Maybe our PCA algorithms are different resulting in different datasets.So I would like to ask you how I should solve this problem, thank you very much.

**Comment by Li-ZK** [comment]
Link: https://pan.baidu.com/s/1bSWcqwIDDCl4bSbLBWrFtQ
Extract code: 906N





发件人：Dave-SUN ***@***.***>
发送日期：2021-06-08 21:42:28
收件人：Li-ZK/MAFN-2021 ***@***.***>
抄送人：LZK ***@***.***>,Comment ***@***.***>
主题：Re: [Li-ZK/MAFN-2021] where main.py (#1)

Yes.I have obtained the data sets after PCA by Matlab,but when I run the program,there are bugs.Maybe our PCA algorithms are different resulting in different datasets.So I would like to ask you how I should solve this problem, thank you very much.

**Comment by Dave-SUN** [comment]
Thank you very much.
